### ISSUE TITLE
iw is missing from the PluralResolver, thus i18next-parse crashes

### ISSUE THREAD

**Issue by kristian-puccio** [i18next/i18next#1494]
## 🐛 Bug Report

The locale code `iw` is missing from the PluralResolver file.
So when you use i18next-parse to extract locales in that language it throws with the error
`Cannot read property 'numbers' of undefined`

## To Reproduce

```js
export function extractTranslations(locales: Array<string>) {
  const srcPath = 'src/**/*.{ts,tsx,js,jsx}';
  gulp
    .src(srcPath)
    .pipe(
      new i18nextParser({
        locales: ['en', 'iw'],
        lowerCaseLng: true,
        output: path.resolve(process.cwd(), 'localisation/$LOCALE/lang.json'),
      })
    )
    .pipe(gulp.dest(process.cwd()));
}
```

## Expected behavior

`iw` should work when using i18next-parse.

## Your Environment

- *runtime version*: i.e. node v14
- *i18next version*: i.e. 19.6.2
    "i18next-parser": "2.1.2",
- *os*: Mac

**Comment by adrai** [comment]
Is iw = he (hebrew)?

**Comment by kristian-puccio** [comment]
Yeah correct

**Comment by adrai** [comment]
Will be deployed with the next version...hope that is ok for you?

If needing it now, you can just use he or you can add rules using i18next.services.pluralResolver.addRule meanwhile...https://github.com/i18next/i18next/blob/master/src/PluralResolver.js#L90

**Comment by kristian-puccio** [comment]
Legend thanks!

**Comment by adrai** [comment]
released with v19.6.3

**Comment by adrai** [comment]
If you like this module don’t forget to star this repo. Make a tweet, share the word or have a look at our https://locize.com to support the devs of this project.

There are many ways to help this project 🙏
